manifests/datasource/oracle.pp

```diff
@@ -5,7 +5,7 @@
 class keycloak::datasource::oracle {
   assert_private()
 
-  $module_source = pick($keycloak::datasource_module_source, 'puppet:///modules/keycloak/database/oracle/module.xml')
+  $jar_filename = pick($keycloak::datasource_jar_filename, 'oracle.jar')
```

Comment:
```suggestion
  $jar_filename = pick($keycloak::datasource_jar_filename, 'ojdbc8.jar')
```

To maintain backwards compatibility the default should remain `ojdbc8.jar`.

Reply:
Hmm, if anybody uses an own module.xml (via parameter `datasource_module_source`) with the filename corrected to the formerly hardcoded `oracle.jar`, this will fail for him/her completely without changing the name via the new parameter `datasource_jar_filename`.
That's not backwards compatible at all, I guess.

Reply:
I think the old behavior worked because the person that contributed the code had `ojdbc8.jar`, not sure how they had that name but that's the name that was used in original pull request.  I think the suggested change is sufficient to maintain backwards compatibility even if it technically might not work. So to use `oracle.jar` you should only need to set `datasource_jar_filename`.  Feel free to update README with Oracle example that uses the code you are using to make things work, that way it's easier for others to see.